src/main/groovy/io/openliberty/tools/gradle/tasks/InstallLibertyTask.groovy

```diff
@@ -136,10 +177,32 @@ class InstallLibertyTask extends AbstractTask {
         return result
     }
 
+    private boolean isMavenCentralConfigured() {
+        boolean found = false
+        project.repositories.find { nextRepo ->
+            if (nextRepo instanceof MavenArtifactRepository) {
```

Comment:
This could be an instance of a MavenArtifactRepository that isn't Maven Central. We should probably check the name/URL of the repo too.